monitoring/understanding-the-monitoring-stack.adoc

```diff
@@ -9,18 +9,26 @@ toc::[]
 include::modules/monitoring-about-cluster-monitoring.adoc[leveloffset=+1]
 
 // Understanding the monitoring stack
-include::modules/monitoring-understanding-the-monitoring-stack.adoc[leveloffset=+1]
+include::modules/about-monitoring-stack-components.adoc[leveloffset=+1]
 include::modules/monitoring-default-monitoring-components.adoc[leveloffset=+2]
 include::modules/monitoring-default-monitoring-targets.adoc[leveloffset=+2]
 include::modules/monitoring-components-for-monitoring-user-defined-projects.adoc[leveloffset=+2]
+
+A cluster administrator can xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack] with the supported configurations. You also have the option to enable xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[monitoring for user-defined projects].
+
 include::modules/monitoring-targets-for-user-defined-projects.adoc[leveloffset=+2]
 
+The {product-title} web console provides xref:../monitoring/accessing-third-party-uis.adoc#accessing-third-party-uis[third-party integrated UIs] such as metrics, alerts, and review of monitoring dashboards. You can access these third-party user interfaces (UIs) either through the web console or the CLI. You can use the **Administrator** or ** Developer** perspective in the {product-title} monitoring dashboard to perform the following tasks:
+
+* xref:../monitoring/managing-metrics.adoc#managing-metrics[Manage metrics]
+* xref:../monitoring/managing-alerts.adoc#managing-alerts[Manage alerts]
+* xref:../monitoring/reviewing-monitoring-dashboards.adoc#reviewing-monitoring-dashboards[Review monitoring dashboards]
+
+
+As a cluster administrator, you can find answers to common problems such as user metrics unavailability and Prometheus consuming a lot of disk space in xref:../monitoring/troubleshooting-monitoring-issues.adoc#troubleshooting-monitoring-issues[troubleshooting monitoring issues].
```

Comment:
Again, this seems not to belong to the section. Would this work as a link under Additional Resources?

Reply:
Created a section where this content is added.